source/includes/fundamentals/code-snippets/CurrentAPI.java

```diff
@@ -19,23 +19,29 @@ public class CurrentAPI {
 
     public static void main(String[] args) throws InterruptedException {
         CurrentAPI c = new CurrentAPI();
+
+        // Call methods on your CurrentAPI instance that demonstrate MongoDB operations
         c.example1();
         c.example2();
 
     }
 
     private void example1() {
+        // Connect to a MongoDB instance with the current API
         // start current-api-example
         MongoClient client = MongoClients.create(URI);
         MongoDatabase db = client.getDatabase(DATABASE);
         MongoCollection<Document> col = db.getCollection(COLLECTION);
+
+        // Find and print a document in your collection to test your connection
```

Comment:
Issue:
Use the indicative mood (verb form that makes a statement).

Suggestion:

I think it's not necessary to mention the "test your connection" part since it doesn't explicitly run any tests. All operations would implicitly test the connection given that they would throw a runtime exception if unable to connect, similar to this operation.

I think the following might be an accurate summary comment that uses the indicative mood:
"Prints the first document retrieved from the collection in JSON format."